packages/curve-ui-kit/src/shared/ui/Metric.tsx

```diff
@@ -83,6 +83,7 @@ type ValueOptions = {
   unit?: Unit | undefined
   /** The number of decimals the value should contain */
   decimals?: number
+  fontColor?: TypographyProps['color']
```

Comment:
I wouldn't place it here actually, because these are common types between both the metric value itself and the notionals. And notional don't support a font color. Would rather make it a prop of `MetricValueProps` directly.

Also personally I'd just call it `color` myself.

And I just noticed `/** Options for any being used, whether` is missing the word 'value' behind 'any', but that was already missing originally.